docs/modules/nonlinearities.rst

```diff
@@ -3,5 +3,20 @@
 
 .. automodule:: lasagne.nonlinearities
 
+.. autosummary::
+
+   sigmoid
```

Comment:
These autosummaries look nice, but since `sigmoid`, `softmax` and `tanh` are directly imported from Theano, it seems that the docstrings from Theano are inserted here. Here's what I see:
```
sigmoid	Generalizes a scalar op to tensors.
softmax	WRITEME
tanh	hyperbolic tangent of a
```

not particularly useful ;) Is there any way to override these without wrapping them in a custom function? If not I guess we should just do that instead, i.e.
```
def sigmoid(x):
    """our custom docstring"""
    return theano.tensor.nnet.sigmoid(x)
```

Would be nice to avoid it though.